examples/Domestic Hot Water/domestic-hot-water-add-pbr.ifcx

```diff
@@ -0,0 +1,164 @@
+      "value": {
+        "dataType": "Object",
+        "objectRestrictions": {
+          "values": {
+            "pbrMetallicRoughness": {
+              "dataType": "Object",
+              "optional": false,
+              "objectRestrictions": {
+                "values": {
+                  "baseColorFactor": {
+                    "dataType": "Array",
+                    "optional": true,
+                    "arrayRestrictions": {
+                      "min": 4,
+                      "max": 4,
+                      "value": {
+                        "dataType": "Real"
+                      }
+                    }
+                  },
+                  "baseColorTexture": {
+                    "dataType": "String",
+                    "optional": true
+                  },
+                  "metallicFactor": {
+                    "dataType": "Real",
+                    "optional": true
+                  },
+                  "roughnessFactor": {
+                    "dataType": "Real",
+                    "optional": true
+                  },
+                  "metallicRoughnessTexture": {
+                    "dataType": "String",
+                    "optional": true
+                  }
+                }
+              }
+            }
+          },
+          "normalTexture": {
+            "dataType": "Object",
+            "optional": true,
+            "objectRestrictions": {
+              "values": {
+                "texture": {
+                  "dataType": "String",
+                  "optional": false
+                },
+                "scale": {
+                  "dataType": "Real",
+                  "optional": true
+                }
+              }
+            }
+          },
+          "occlusionTexture": {
+            "dataType": "Object",
+            "optional": true,
+            "objectRestrictions": {
+              "values": {
+                "texture": {
+                  "dataType": "String",
+                  "optional": false
+                },
+                "strength": {
+                  "dataType": "Real",
+                  "optional": true
+                }
+              }
+            }
+          },
+          "emissiveTexture": {
+            "dataType": "String",
+            "optional": true
+          },
+          "emissiveFactor": {
+            "dataType": "Array",
+            "optional": true,
+            "arrayRestrictions": {
+              "min": 3,
+              "max": 3,
+              "value": {
+                "dataType": "Real"
+              }
+            }
+          },
+          "alphaMode": {
+            "dataType": "Enum",
+            "optional": true,
+            "enumRestrictions": {
+              "options": [
+                "OPAQUE",
+                "MASK",
+                "BLEND"
+              ]
+            }
+          },
+          "alphaCutoff": {
+            "dataType": "Real",
+            "optional": true
```

Comment:
This and many other fields could be further elaborated, for example, alphaCutoff should also have (based on https://registry.khronos.org/glTF/specs/2.0/glTF-2.0.html#appendix-a-json-schema-reference):
```
"min?": 0.0,
"default": 0.5,
```
(not sure if that's the way we will be capturing the restrictions in the schema)